src/hydroshoot/irradiance.py

```diff
@@ -38,11 +38,11 @@ def local2solar(local_time, latitude, longitude, time_zone, temperature=25.):
 
     time_zone = timezone(time_zone)
     local_time = time_zone.localize(local_time)
-    date_utc = local_time.astimezone(utc)
+#    date_utc = local_time.astimezone(utc)
     datet = date_range(local_time, local_time)
-    lst = ephemeris(datet, latitude, longitude, temperature).solar_time.values[0]
+    solar_time = ephemeris(datet, latitude, longitude, temperature).solar_time.values[0]
 
-    return date_utc, lst
+    return solar_time
```

Comment:
is solar_time a tuple of 2 elements?
Otherelse, modify the documentation. It indicates that the function returns two elemnts